### PR TITLE
Added ROS messages to Python and removed nan timestamps from numpy data and plots.

### PR DESCRIPTION
**Pull request by adamshapiro0**

### CODE REVIEW

**Comment by adamshapiro0** [comment]
@jetsonhacks FYI pulled this from your fork. Thanks for the contribution!

File: python/fusion_engine_client/messages/ros.py [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+    @brief ROS `GPSFix` message (MessageType::ROS_GPS_FIX).
+    """
+    MESSAGE_TYPE = MessageType.ROS_GPS_FIX
+
+    _FORMAT = '< ddd ddd ddd d ddddd dddddddddd 9d B 3B'
+    _SIZE: int = struct.calcsize(_FORMAT)
+
+    def __init__(self):
+        self.p1_time = Timestamp()
+        # WGS-84 Geodetic Position
```
Comment (anathan) [inline python/fusion_engine_client/messages/ros.py:101]: These are not necessarily WGS84

**Review by anathan — APPROVED** [review]
little nit about the datum stuff.

File: python/fusion_engine_client/messages/ros.py (hunk 1 above)
Comment (adamshapiro0) [reply python/fusion_engine_client/messages/ros.py:101]: Yeah, the ROS message definition doesn't actually specify. I'll fix this comment.

**Pull request merged**